### ISSUE TITLE
Wallet wont sync.

### ISSUE THREAD

**Issue by StianNOR** [torus-economy/torus-core#36]
Wallet wont sync, getting error for trying to addnode IP onetry
No wallet.conf to add nodes, Wallet say 1 connection and no sync.

**Comment by sskender** [comment]
You need at least three connections in order to successfully sync and stake. Also, you do not need to add IP addresses of seed nodes manually, they are already hardcoded into the application binary.

**Comment by StianNOR** [comment]
Yes i see it started to sync now. i did install latest version first. nothing happend.
I dident get up the sync status whit bloch remaining. installed version 1.0.2 that one showed the status and started to sync.
Will update after synced 100%, dunno if its a bug for brand new wallets or whats going on.

**Comment by sskender** [comment]
Latest version may not be fully stable and blockchain is still in development. Would be great if you could keep us up to date. 

EDIT: Which system are you using? Windows, Linux, Docker?

**Comment by sskender** [comment]
It looks like the network is not fully in sync with v1.0.3, so please add the following into TORUS.conf file:

```
addnode=154.56.63.68
addnode=154.56.63.66
addnode=153.92.223.18
addnode=207.180.236.165
```

This should resolve problems with syncing.
